### DISCUSSION TITLE
Display OBS Scenes on Mouse Hover

### DISCUSSION THREAD

**Discussion by joaochris** [sircharlo/meeting-media-manager#2809] [Ideas] [answered]
Hello @sircharlo

First of all, thank you for your work on this great update 🙂

It was mentioned in the other repository about having OBS scenes always visible in the program, like in the previous version.

However, to keep the current app's design, would it be possible to implement this, but instead of always showing them, make the scenes appear when hovering over them with the mouse?

This way, they would be just a click away, making it easier for those who need to switch between multiple scenes in OBS. We use several scenes here, and this would be very helpful 😄.

**Comment by sircharlo** [comment]
This was already being discussed. We'll see what we can do.

**Reply by joaochris** [reply]
@sircharlo I saw the video about what the floating button you created would look like, it was wonderful!

**Reply by sircharlo** [reply]
@ux-git  deserves the credit for that one :)

**Reply by sircharlo** [reply, marked as answer]
Implemented in c31f047c9d98a0ee1f91246d981579656c372396. Looking forward to everyone's feedback!

**Comment by mtdvlpr** [comment]
> [!IMPORTANT]
> You might have to install the new version manually instead of automatically, due to a bug that's been fixed for upcoming releases.
